<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Janela</class>
 <widget class="QWidget" name="Janela">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FAMA</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>607</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_preProcess">
    <attribute name="title">
     <string>Pré-processamento</string>
    </attribute>
    <widget class="QComboBox" name="comboBox_corpus">
     <property name="geometry">
      <rect>
       <x>107</x>
       <y>9</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAlphabetically</enum>
     </property>
     <item>
      <property name="text">
       <string>Escolha o Corpus</string>
      </property>
     </item>
    </widget>
    <widget class="QGroupBox" name="groupBox_relatorioGeral">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>36</y>
       <width>296</width>
       <height>71</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>71</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Corpus</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_relatorio">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>       Nome:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEdit_relatorio">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="text">
         <string>Nenhum</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_instancias">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>  Instâncias:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_instancias">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="text">
         <string>Nenhum</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_atributos">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Atributos:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEdit_atributos">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="text">
         <string>Nenhum</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_atributoSelecionado">
     <property name="geometry">
      <rect>
       <x>311</x>
       <y>36</y>
       <width>283</width>
       <height>265</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Atributo selecionado</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="label_nome">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>25</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>     Nome:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nome">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>25</y>
        <width>73</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>Nenhum</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_tipo">
      <property name="geometry">
       <rect>
        <x>154</x>
        <y>25</y>
        <width>28</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tipo:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_tipo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>199</x>
        <y>25</y>
        <width>73</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>Nenhum</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_distintos">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>47</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  Distintos:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_distintos">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>47</y>
        <width>73</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>Nenhum</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_unicos">
      <property name="geometry">
       <rect>
        <x>154</x>
        <y>47</y>
        <width>39</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Únicos:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_unicos">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>199</x>
        <y>47</y>
        <width>73</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>Nenhum</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_estatistica">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>69</y>
        <width>261</width>
        <height>185</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>130</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>26</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>17</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>17</number>
      </attribute>
      <column/>
      <column/>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_atributos">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>113</y>
       <width>296</width>
       <height>215</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Atributos</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableWidget_atributos">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>50</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>17</number>
        </attribute>
        <column/>
        <column/>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QToolButton" name="toolButton_construtor">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_abrir">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>92</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>101</width>
       <height>21</height>
      </size>
     </property>
     <property name="font">
      <font>
       <italic>false</italic>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Abrir arquivo...</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_validacao">
    <attribute name="title">
     <string>Classificação</string>
    </attribute>
    <widget class="QComboBox" name="comboBox_metodo">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAlphabetically</enum>
     </property>
     <item>
      <property name="text">
       <string>Escolha o Método</string>
      </property>
     </item>
    </widget>
    <widget class="QGroupBox" name="groupBox_opcoesTeste">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>261</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Opções de Teste</string>
     </property>
     <widget class="QRadioButton" name="radioButton_treino">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Utilizar conjunto de treino</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_teste">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Fornecer conjunto de teste</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_teste">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Abrir</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_kDobras">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Validação cruzada</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_kDobras">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>46</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Dobras</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_kDobras">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_divisao">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Porcentagem de divisão</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_divisao">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_kDobras_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_opcoes">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Opções</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_start">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>tableWidget_atributos</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>Janela</receiver>
   <slot>atributoSelecionado(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_corpus</sender>
   <signal>activated(int)</signal>
   <receiver>Janela</receiver>
   <slot>habilitarBotao(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_construtor</sender>
   <signal>clicked()</signal>
   <receiver>Janela</receiver>
   <slot>definirParametros()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_abrir</sender>
   <signal>clicked()</signal>
   <receiver>Janela</receiver>
   <slot>abrirArquivo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_metodo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Janela</receiver>
   <slot>escolherClassificador(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>abrirArquivo()</slot>
  <slot>habilitarBotao(int)</slot>
  <slot>atributoSelecionado(int,int)</slot>
  <slot>definirParametros()</slot>
  <slot>escolherClassificador(int)</slot>
 </slots>
</ui>
